shower.js

```diff
@@ -14,12 +14,56 @@ window.shower = (function(window, document, undefined) {
 		slides = [],
 		progress = [],
 		timer,
-		isHistoryApiSupported = ! ! (window.history && window.history.pushState);
+		MODES = {
```

Comment:
No, I don't have. Google codestyle for enums looks good. I will rename var to UpperCamelCase.